providers/powerdns/powerdnsProvider.go

```diff
@@ -23,6 +23,7 @@ var features = providers.DocumentationNotes{
 	providers.CanUseLOC:              providers.Unimplemented("Normalization within the PowerDNS API seems to be buggy, so disabled", "https://github.com/PowerDNS/pdns/issues/10558"),
 	providers.CanUseNAPTR:            providers.Can(),
 	providers.CanUsePTR:              providers.Can(),
+	providers.CanUseSOA:              providers.Can(),
```

Comment:
```suggestion
	providers.CanUseSOA:              providers.Can("Please make sure that SOA-EDIT-API is deactivated if you want to use it", "https://doc.powerdns.com/authoritative/dnssec/operational.html#possible-soa-edit-values"),
```

Reply:
Ah! I totally didn't think of that part. Good catch!